meteor-app/private/members2.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
I thought you pulled me up about not putting personal details in the git repo (which is public). Just putting them in the private folder doesn't fix it. 
It may stop Meteor from delivering it to the UI, but it's still in the code base. We need to work out a way to not have it here

Reply:
There are quite a few conflicts below as well - not sure how this happened - something else for you to look at